c_glib/arrow-glib/compute.cpp

```diff
@@ -5192,3 +5301,21 @@ garrow_round_to_multiple_options_get_raw(GArrowRoundToMultipleOptions *options)
   return static_cast<arrow::compute::RoundToMultipleOptions *>(
     garrow_function_options_get_raw(GARROW_FUNCTION_OPTIONS(options)));
 }
+
+
+GArrowUTF8NormalizeOptions *
+garrow_utf8_normalize_options_new_raw(
+  const arrow::compute::Utf8NormalizeOptions *arrow_options)
+{
+  auto options = g_object_new(GARROW_TYPE_UT_F8_NORMALIZE_FORM,
```

Comment:
I don't know why i can use 'GARROW_TYPE_UTF8_NORMALIZE_FORM' variable.

When compiling, I get the following error: ``use of undeclared identifier 'GARROW_TYPE_UTF8_NORMALIZE_FORM``

Could you tell me how to fix it?

Reply:
`glib-mkenums` guesses wrong identifier.
I added an annotation to specify correct identifier explicitly.